src/main/java/de/tum/in/www1/artemis/service/LectureUnitProcessingService.java

```diff
@@ -130,6 +132,10 @@ private boolean isBreakSlide(String slideText) {
         return slideText.contains("Break") || slideText.contains("Pause");
     }
 
+    private boolean isSolutionSlide(String slideText) {
+        return slideText.contains("Example solution") || slideText.contains("Example solution (with comments)") || slideText.contains("Example solution: model");
```

Comment:
Why not offer the user a text box next to the option of removing the slides?
This way the user can choose the text themselves and it works for all lectures in all languages

Reply:
Thats a good idea. @krusche What do you think of this suggestion?

Reply:
@JohannesStoehr But i still think that we should merge this PR and then apply your suggestion in a follow up for solution slides and break slides.

Reply:
yes, this would be a good idea for a follow-up